spec/services/message_employee_matcher_spec.rb

```diff
@@ -2,10 +2,18 @@
 
 describe MessageEmployeeMatcher do
   describe "#run" do
+    before do
```

Comment:
yes! - for more context on the thinking behind this, this is a good one: https://robots.thoughtbot.com/lets-not